hazelcast/src/main/java/com/hazelcast/map/impl/MapSplitBrainHandlerService.java

```diff
@@ -60,13 +60,15 @@
 
 class MapSplitBrainHandlerService implements SplitBrainHandlerService {
 
-    private final int partitionCount;
-    private final ILogger logger;
-    private final NodeEngine nodeEngine;
-    private final OperationService operationService;
-    private final IPartitionService partitionService;
-    private final MapServiceContext mapServiceContext;
-    private final MergePolicyProvider mergePolicyProvider;
+    static final int TIMEOUT_FACTOR = 500;
```

Comment:
The `TIMEOUT_FACTOR` seems to be duplicated now. Could we leave it in the Runnable, please? Matko already prototyped to have an abstract Merger class, which would have it in a single place eventually.